start_cerberus.py

```diff
@@ -259,6 +261,26 @@ def main(cfg):
                     logging.info("Route url: %s" % route)
                 logging.info("")
 
+            # Aggregate the status and publish it
+            cerberus_status = watch_nodes_status and watch_namespaces_status \
+                and watch_cluster_operators_status and server_status
+
+            if custom_checks:
+                if iteration == 1:
```

Comment:
what does iteration == 1 mean ?  does that mean custom checks are run only once ?. Or in general does custom checks are treated as how other checks are treated in cerberus ?

Reply:
Custom checks are executed in every iteration. In the first iteration (iteration == 1), all the custom checks are imported.